src/Libraries/CoreNodeModels/Input/IntegerSlider.cs

```diff
@@ -334,6 +335,18 @@ protected override bool UpdateValueCore(UpdateValueParams updateValueParams)
             return base.UpdateValueCore(updateValueParams);
         }
 
+        private void UpdateNodeInfo(string value)
+        {
+            if (IsValueInt64(value))
+            {
+                ClearInfoMessages();
+            }
+            else
+            {
+                Info(Resources.IntegerSliderInfoMessage);
```

Comment:
This should be changed to:
```
this.Infos.Add(new Info(Resources.IntegerSliderInfoMessage));
```

Reply:
Created this Info() as we have a similar Error() and Warning() on NodeModel object.